pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -32,8 +32,8 @@ let
   # "${mktplcRef.publisher}.${mktplcRef.name}".
   #
   baseExtensions = self: lib.mapAttrs (_n: lib.recurseIntoAttrs)
-    {
-      _1Password.op-vscode = buildVscodeMarketplaceExtension {
+    ({
```

```suggestion
    ({
```
Why `(` is necessary here?

---

it isn’t, thanks for noticing